src/main/frontend/components/SiteShow.js

```diff
@@ -96,18 +134,20 @@ const SiteShow = (props) => {
     />
   }
 
-  const { id, name, description, imgUrl, url } = site
-
   return (
     <div>
       <a href={url}><h1>{name}</h1></a>
       <img src={imgUrl} />
       <a href={url} target="_blank"><p>Visit the Site</p></a>
       <p><strong>Description:</strong> {description}</p>
+      <div>
+      <button type ="button" onClick = {handleDeleteClick} >Delete Site </button>
+      </div>
       {successMessageTag}
       <button type="button" className="button" onClick={handleFormButtonClick}>Add A Review</button>
       {newReviewForm}
       {reviewTiles}
+      {redirect}
```

Comment:
rather than put the redirect on the existing page, we should `return <Redirect...>` up on line 107 so that our component doesn't have to put in the work of loading all of this before redirecting!

Please also fix indentation on lines 105-114 :)